main/lsp/LSPTypechecker.h

```diff
@@ -148,10 +158,13 @@ class LSPTypechecker final {
  * Provides lambdas with a set of operations that they are allowed to do with the LSPTypechecker.
  */
 class LSPTypecheckerDelegate {
-    WorkerPool &workers;
     LSPTypechecker &typechecker;
 
 public:
+    /** The WorkerPool on which work will be performed. If the task is multithreaded, the pool will contain multiple
+     * worker threads. */
+    WorkerPool &workers;
```

Comment:
Some of the new tasks I added need direct access to the WorkerPool to distribute work amongst worker threads. They don't involve `LSPTypechecker`, so it didn't make sense to add a layer of delegation to them.